CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Metadata backup] Add a timeout for the pool metadata backups [#4657](https://github.com/vatesfr/xen-orchestra/issues/4657) (PR [#4666](https://github.com/vatesfr/xen-orchestra/pull/4666))
```

Comment:
What does this change for users?

Can they configure the timeout for each job?

Reply:
No but they will know that a timeout is added to the metadata backup and their job will not be stuck again.

Reply:
IMHO it's not clear enough for a user.